packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -112,6 +124,17 @@ class GestureArenaManager {
   }
 
   /// Prevents the arena from being swept.
+  ///
+  /// Typically, a winner is chosen in an arena after all the other
+  /// [PointerUpEvent] processing by [sweep]. If a recognizer wishes to delay
```

Comment:
put a comma after "processing" because otherwise it looks like the processing itself is done by "sweep", as opposed to the choosing of a winner.